.github/workflows/sync.yml

```diff
@@ -0,0 +1,68 @@
+name: Sync Simple Subtree
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs every day at midnight UTC
+  workflow_dispatch:    # Allows manual triggering
+
+jobs:
+  sync-subtree:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository with full Git history.
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Required for subtree operations to work correctly
+
+      # Configure Git with a default user for commits made by this action.
+      - name: Configure Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
@filipelautert we'll likely need to substitute this with a user that has push access to the repo.  This will likely fail at first.

Reply:
actually since both repos are public this should work fine in the demo.